src/js/playwright.config.ts

```diff
@@ -0,0 +1,29 @@
+import { defineConfig, devices } from "@playwright/test";
+import path from "node:path";
+
+export default defineConfig({
+  testDir: "test/integration",
+  fullyParallel: true,
+  webServer: {
+    command: "npx http-server -p 3000",
```

Comment:
I'm not sure whether it is good to fix specific port, but in order to set a dynamic port available on the machine, the code could be verbose.

Feel free to give me feedback!

Reply:
I think it is okay to use the fixed port. We can fix it when we have an issue later. Thanks!